projects.go

```diff
@@ -228,6 +228,12 @@ type ListProjectsOptions struct {
 	MinAccessLevel           *AccessLevelValue `url:"min_access_level,omitempty" json:"min_access_level,omitempty"`
 	WithCustomAttributes     *bool             `url:"with_custom_attributes,omitempty" json:"with_custom_attributes,omitempty"`
 	WithProgrammingLanguage  *string           `url:"with_programming_language,omitempty" json:"with_programming_language,omitempty"`
+	WikiChecksumFailed       *bool             `url:"wiki_checksum_failed,omitempty" json:"wiki_checksum_failed,omitempty"`
+	RepositoryChecksumFailed *bool             `url:"repository_checksum_failed,omitempty" json:"repository_checksum_failed,omitempty"`
```

Comment:
I know you didn't add it there, but could you move `MinAccessLevel` below `RepositoryChecksumFailed` to keep the fields inline with the ordering in the docs? Thanks!

Reply:
updated all ordering to match the docs